### DISCUSSION TITLE
Naming convention for input and output of Controllers

### DISCUSSION THREAD

**Discussion by mehdihadeli** [kgrzybek/modular-monolith-with-ddd#208] [Ideas] [answered]
Hi,
I've seen in your controllers for sending data as dto you used a naming convention like `**Request` for example [here](
https://github.com/kgrzybek/modular-monolith-with-ddd/blob/37de06228f3f6f49de20a510d5349c784903b01f/src/API/CompanyName.MyMeetings.API/Modules/Meetings/Meetings/MeetingsController.cs#L58) but for query a data and return it from controller you used dto suffix for it `**Dto` for example [here](https://github.com/kgrzybek/modular-monolith-with-ddd/blob/37de06228f3f6f49de20a510d5349c784903b01f/src/API/CompanyName.MyMeetings.API/Modules/Meetings/Meetings/MeetingsController.cs#L49). It is not better we use same structure for their naming convention  for example `**Request` for our input and `**Reposne` for our output or using `**Dto` suffix for both input data and output data.

**Comment by kgrzybek** [comment, marked as answer]
I always try to name an object which is deserialized from body request as `**Request` to distinguish it from the `**Command`. 

If you like naming returned objects as `**Response` i think it is good too. Important thing is that it is a *data structure* from the conceptual perspective, this is main reason why I call it *Dto*

**Reply by mehdihadeli** [reply]
Thanks for your answer
